### PR TITLE
Add tooltip documentation for default false, on autoClose param, of useToast

### PR DESCRIPTION
**Pull request by cristidumitrescutotalsoft**

### CODE REVIEW

**Comment by cristidumitrescutotalsoft** [comment]
https://github.com/osstotalsoft/rocket-ui-ts/issues/124#issue-2465079897

File: src/components/feedback/Toast/types.ts [hunk 1]
```diff
@@ -9,3 +9,12 @@ export interface ToastContainerProps extends Omit<ReactToastifyProps, 'transitio
...
   transitionType?: 'Slide' | 'Bounce' | 'Zoom' | 'Flip'
 }
+
+export interface ToastifyToastOptionsCustom extends ToastifyToastOptions {
+  /**
+   * Set the delay in ms to close the toast automatically.
+   * Use `false` to prevent the toast from closing.
+   * `Default: false`
+   */
+  autoClose?: number | false
```
Comment (DCosti) [inline src/components/feedback/Toast/types.ts:19]: `autoClose` property should be moved to `ToastContainerProps`. We don't need another interface just to override the `autoClose` prop

Comment (DCosti) [reply src/components/feedback/Toast/types.ts:19]: Inside `ReactToastifyProps` , the interface we extends, you already have the `autoClose` property. You just need to add it in the `Omit` section and then set the default

File: src/components/feedback/Toast/useToast.ts [hunk 2]
```diff
@@ -39,9 +39,9 @@ const useToast = () => {
         [T, always(Slide)]
       ])
 
-      const options: ToastifyToastOptions = {
+      const options: ToastifyToastOptionsCustom = {
         ...restOptions,
-        autoClose: autoClose || false,
+        autoClose: autoClose,
```
Comment (DCosti) [inline src/components/feedback/Toast/useToast.ts:44]: is the same name. just `autoClose` should be enough

File: src/components/feedback/Toast/types.ts (hunk 1 above)
Comment (cristidumitrescutotalsoft) [reply src/components/feedback/Toast/types.ts:20]: modified it accordingly in the latest commit

File: src/components/feedback/Toast/useToast.ts (hunk 2 above)
Comment (cristidumitrescutotalsoft) [reply src/components/feedback/Toast/useToast.ts]: modified it accordingly in the latest commit

File: src/components/feedback/Toast/useToast.ts [hunk 3]
```diff
@@ -39,9 +39,9 @@ const useToast = () => {
         [T, always(Slide)]
       ])
 
-      const options: ToastifyToastOptions = {
+      const options: ToastContainerProps = {
         ...restOptions,
-        autoClose: autoClose || false,
+        autoClose,
```
[the remark this answers is not in the document]
Comment (cristidumitrescutotalsoft) [reply src/components/feedback/Toast/useToast.ts]: I have removed it in the last commit

**Review by DCosti — APPROVED** [review]

**Pull request merged**